README.adoc

```diff
@@ -91,13 +91,13 @@ tekton-dashboard   tekton-dashboard-tekton-pipelines.apps.kabanero.os.fyre.ibm.c
 Use the Jenkinsfile template from the reference section of this guide, and change the Docker Hub source to your own image repository. Push this Jenkinsfile to your GitHub repository in the root location.
 
 . **Create a deploy file**:
-On your local environment where Appsody is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
+On your local environment where your development application is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
 +
 ----
 appsody-deploy –-generate-only
 ----
 +
-Other tools, including Jenkins, can use this file for application deployment. Commit and push this file to the root of your project. Jenkins creates the `AppsodyApplication` resource in the OpenShift cluster and uses Appsody to deploy the application by using the `app-deploy.yaml` file.
+Other tools, including Jenkins, can use this file for development application deployment. Commit and push this file to the root of your project. Jenkins creates the `AppsodyApplication` resource in the OpenShift cluster and uses your application to deploy the application by using the `app-deploy.yaml` file.
 
 . **Create a Jenkins project and pipeline**:
 Create a Jenkins (Ephemeral) instance by running the following command:
```

Comment:
I've always thought these steps are just craziness.  This guide should show how to trigger Jenkin's from a Tekton pipeline task as an example; not some manual process for kicking off the process.

Reply:
agreed.  next rev maybe?